pom.xml

```diff
@@ -76,7 +76,7 @@
 
         <assertj.version>3.25.3</assertj.version>
         <bouncycastle.version>1.78</bouncycastle.version>
-        <commons-compress.version>1.26.1</commons-compress.version>
+        <commons-compress.version>1.28.0-SNAPSHOT</commons-compress.version>
```

Comment:
Does this really need the snapshot version? As surely, the project can't really release with a snapshot dependency?

Reply:
You can consider it a work-in-progress until the 1.28.0 release.

It does need the latest SNAPSHOT, because I requested a couple features from upstream in order to support setting the flags (i.e., there is no way to set the compressor options in most of the compressors until the next release).

Reply:
- XZ options: https://github.com/apache/commons-compress/commit/5829240bfd3531639767e40ddf9608b84d3b235c
- LZMA options: https://github.com/apache/commons-compress/commit/55ecb7f1b8f58408c525fef742dcf27d51efebf3
- Zstd options: https://github.com/apache/commons-compress/pull/666

Reply:
OK, commons-compress 1.28.0 has been released.